packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/test/lib_pin_code/views/mocks/pin_biometrics_local_datasource_mock.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:{{project_name}}/lib_pin_code/data_source/pin_biometrics_local_data_source.dart';
+
+import 'pin_biometrics_local_datasource_mock.mocks.dart';
+
+@GenerateMocks([PinBiometricsLocalDataSource])
+PinBiometricsLocalDataSource pinBiometricsLocalDataSourceMockFactory() {
```

Comment:
Please parametrise the method such as `{areBiometricsEnabled: true}` so that you can use in the tests with or without biometrics being enabled. Please use this approach for all mock factory methods you have implemented

Reply:
@StanevPrime does this need additional change?